### ISSUE TITLE
Unciv not launching

### ISSUE THREAD

**Issue by meyou69** [yairm210/Unciv#12379]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Game Version

4.14.1

### Describe the bug

When launching the game, the game does not load and exits immediately.

Not even a window appears at all on my desktop.

Please note: I am specifically talking about the flatpak version of the game 

### Steps to Reproduce

1. Load the game page in Discover
2. Press the Launch button (the game needs to be installed beforehand)
3. Wait a couple of seconds
4. Nothing happens

Alternatively, you can also try to launch the game from the command line, where the following error appears:

```
(base) alex@fedora:~$ /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=unciv io.github.yairm210.unciv
[LWJGL] GLFW_PLATFORM_ERROR error
        Description : Wayland: Failed to connect to display
        Stacktrace  :
                org.lwjgl.glfw.GLFW.glfwInit(GLFW.java:1109)
                com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.initializeGlfw(Lwjgl3Application.java:89)
                com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration.getPrimaryMonitor(Lwjgl3ApplicationConfiguration.java:281)
                com.unciv.app.desktop.DesktopScreenMode$Companion.getMaximumWindowBounds$desktop$default(DesktopDisplay.kt:85)
                com.unciv.app.desktop.DesktopLauncher.main(DesktopLauncher.kt:82)
Exception in thread "main" com.badlogic.gdx.utils.GdxRuntimeException: Unable to initialize GLFW
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3Application.initializeGlfw(Lwjgl3Application.java:90)
        at com.badlogic.gdx.backends.lwjgl3.Lwjgl3ApplicationConfiguration.getPrimaryMonitor(Lwjgl3ApplicationConfiguration.java:281)
        at com.unciv.app.desktop.DesktopScreenMode$Companion.getMaximumWindowBounds$desktop$default(DesktopDisplay.kt:85)
        at com.unciv.app.desktop.DesktopLauncher.main(DesktopLauncher.kt:82)
```

### Operating System

Linux

### Additional Information

The operating system that I was using was Fedora KDE Spin version 40. Since then, I have upgraded to Fedora KDE Spin version 41. The bug reproduces on both of them, equally

**Comment by FilipRank** [comment]
I have the same issue the 1.14.1 jar on my Mac. 1.14.0 works without issues.

**Comment by nintyfan** [comment]
Same error on Wayland.

Temporary solution: Start new terminal emulator window and type:
```
unset WAYLAND_DISPLAY
flatpak run io.github.yairm210.unciv && exit
```

**Comment by meyou69** [comment]
> Same error on Wayland.
> 
> Temporary solution: Start new terminal emulator window and type:
> 
> ```
> unset WAYLAND_DISPLAY
> flatpak run io.github.yairm210.unciv && exit
> ```

I can confirm that this solution worked for me.

A long term solution is still needed, though

**Comment by yairm210** [comment]
Looks like this is a problem with the latest LibGDX version which I just updated
https://github.com/libgdx/libgdx/issues/7495
I'll revert this change in next version

**Comment by yairm210** [comment]
Resolved with 0c6ae89b
